### PR TITLE
GA and privacy

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: views/footer.hjs [hunk 1]
```diff
@@ -12,3 +12,26 @@
...
+	    'clientId': sessionStorage.getItem(GA_SESSION_STORAGE_KEY)
+	   });
+	   ga(function(tracker) {
+	    sessionStorage.setItem(GA_SESSION_STORAGE_KEY, tracker.get('clientId'));
+	   });
+   }
+	ga('set', 'anonymizeIp', true);
+	ga('send', 'pageview');
+}
+</script>
```
Comment (gaborcsardi) [inline views/footer.hjs]: Can you pls add a newline to the end of the file?

File: views/terms.hjs [hunk 2]
```diff
@@ -41,6 +41,17 @@
 	    purposes. We do not give out the check output to anyone,
 	    except for the package maintainer, as declared in the
 	    DESCRIPTION file of the package.</p>
+	    
+	   <p>We use <a href="https://analytics.google.com/analytics/web/">Google Analytics</a> to log data and ultimately improve the website, with the following settings:
```
Comment (gaborcsardi) [inline views/terms.hjs]: Can you pls break long lines at ~75 character? Thanks.

**Comment by maelle** [comment]
@gaborcsardi I'll let you merge this if you're happy with the changes.

**Comment by gaborcsardi** [comment]
Thanks!

**Comment by gaborcsardi** [comment]
Will deploy, once back in my office, in about 30min.

**Comment by gaborcsardi** [comment]
OK, deployed.

**Comment by maelle** [comment]
Thank you!

**Pull request merged**